atomic_reactor/util.py

```diff
@@ -947,3 +955,21 @@ class LabelFormatter(string.Formatter):
     """
     def get_field(self, field_name, args, kwargs):
         return (self.get_value(field_name, args, kwargs), field_name)
+
+
+def do_retries(client_statuses=HTTP_CLIENT_STATUS_RETRY,
```

Comment:
I'd prefer if this was called something more descriptive or specific. It sounds more generic than it is. How about `get_retrying_requests_session`?

Reply:
yeah now it isn't as generic as it was before :)

Reply:
Since it returns `requests.Session` lets call it `get_requests_session` or similar, as later on we might add some more requests features here